pyrasterframes/src/main/python/docs/supervised-learning.pymd

```diff
@@ -0,0 +1,205 @@
+# Supervised Machine Learning
+
+In this example we will demonstrate how to fit and score an unsupervised learning model with a sample of Sentinel-2 data and hand-drawn vector labels over different [land cover](https://en.wikipedia.org/wiki/Land_cover) types.
```

Comment:
Should be "supervised" not "unsupervised"

Reply:
@vpipkt Fix this before merge?

Reply:
Great catch .... 🤷‍♂